### ISSUE TITLE
OpenSSL 1.1.1g - ssl.SSLError: [SSL: CA_MD_TOO_WEAK] ca md too weak (_ssl.c:3503)

### ISSUE THREAD

**Issue by davepee** [ma1co/Sony-PMCA-RE#211]
Hello, on Ubuntu 18.04 with OpenSSL 1.1.1g I'm still getting this error:

`sl.SSLError: [SSL: CA_MD_TOO_WEAK] ca md too weak (_ssl.c:3503)`

Sony a5000

Thank you.

**Comment by nogkaha** [comment]
I'm having a similar issue with SSL, I'm using sony a6000 and the following system info:

Ubuntu 20.04.1 LTS
OpenSSL 1.1.1f 
Python 3.8.5

Output when I'm trying to install TimeLapse app 
```
No native drivers available
Using drivers libusb-MSC, libusb-MTP
Looking for Sony devices

Querying MTP device
Sony Corporation ILCE-6000 is a camera in MTP mode

Model:              ILCE-6000
Product code:       0032826810
Serial number:      03655193
Firmware version:   3.10

No native drivers available
Using drivers libusb-MSC, libusb-MTP
Looking for Sony devices

Querying MTP device
Sony Corporation ILCE-6000 is a camera in MTP mode

Switching to app install mode
Waiting for camera to switch...
Traceback (most recent call last):
  File "./pmca-gui.py", line 76, in do
    installCommand(appPackage=app.package)
  File "/home/noga/projects/sony_installer/ma1co-Sony-PMCA-RE-2c24d4b/pmca/commands/usb.py", line 294, in installCommand
    installApp(device, apkFile, appPackage, outFile, local)
  File "/home/noga/projects/sony_installer/ma1co-Sony-PMCA-RE-2c24d4b/pmca/commands/usb.py", line 61, in installApp
    with ServerContext(LocalMarketServer(certFile, config.officialServer)) as server:
  File "/home/noga/projects/sony_installer/ma1co-Sony-PMCA-RE-2c24d4b/pmca/marketserver/server.py", line 68, in __init__
    self.socket = ssl.wrap_socket(self.socket, server_side=True, ssl_version=ssl.PROTOCOL_TLSv1, certfile=certFile)
  File "/usr/lib/python3.8/ssl.py", line 1402, in wrap_socket
    context.load_cert_chain(certfile, keyfile)
ssl.SSLError: [SSL: CA_MD_TOO_WEAK] ca md too weak (_ssl.c:4022)
```

**Comment by ma1co** [comment]
This should be fixed with 9ab9dbe0cddb24d160633277161a3a870aa0b9ab.
